### PR TITLE
ComboQuery

### PR DESCRIPTION
**Pull request by themandalore**

### CODE REVIEW

**Review by oraclown — APPROVED** [review]
Thanks @themandalore 

This is just for a combination of queries with identical response types, right? Bc you couldn't have a ComboQuery that included a spot price query and another that required a string response, no? Maybe should include that in the description, or we assume the Sol devs would just already know this..

**Comment by oraclown** [comment]
> > required
> 
> why couldn't you? You would just return an array with the spot price answer, and then the string...
> 
> e.g. for the answer ( abi.encode([1, "myanswer"]) )

Oh, I see. Yeah, if decoding responses with multiple types on the Sol side is fine, then I think it all looks good

**Comment by tkernell** [comment]
I think this looks good. One possible improvement: why even include the queryIds array? We could save gas costs and only include the queryData array

File: types/ComboQuery.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- It is the reporters responsibility to view the dispute considerations for each queryId in the combo datatype
+
+
+## Example
+
+In the example query:  
+
+```
+abi.encode("ComboQuery", abi.encode(
+            [abi.encode("SpotPrice", abi.encode("ETH","USD")),
```
Comment (tkernell) [inline types/ComboQuery.md:81]: SpotPrice args should be lower case: SpotPrice(eth, usd)

Comment (themandalore) [reply types/ComboQuery.md]: fixed

**Comment by tkernell** [comment]
Looks good!

**Pull request merged**